database/articles.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const expressApp = express();
+const bodyParser = require('body-parser');
+const jsonParser = bodyParser.json();
+
+let articles = [];
+
+const getAllArticles = () => {
+  return articles;
+};
+
+const postArticle = (req, res) => {
+  if (req.body) {
+    articles.push(req.body);
+    res.send({ success: true });
+    console.log(articles);
+  } else {
+    res.send({ success: false });
```

Comment:
I like to simplify as much as possible and only use `else` statements when needed in favor of `return`. Same function could be written like:

```
if (!req.body) res.send({ success: false });

articles.push(req.body);
res.send({ success: true });
```